compareECLFiles.cmake

```diff
@@ -742,11 +743,19 @@ add_test_compare_restarted_simulation(CASENAME spe9
                                       ABS_TOL ${abs_tol_restart}
                                       REL_TOL ${rel_tol_restart}
                                       TEST_ARGS --sched-restart=false)
+
+# The dynamic MSW data is not written to /read from the restart file
+# We therefore accept significant deviation in the results.
+# Note also that we use --sched-restart=true since some necessary
+# MSW info is still lacking in the restart file.
+set(rel_tol_restart_msw 2e2)
```

Comment:
This should be abs_tol I think. A relative tolerance of 20000% is a bit much...

Reply:
It worries my that I need to relax the absolute tolerance to 2e2 for this. Locally it runs through without any adjustments. @atgeirr Maybe you could test this (without the last commit with the adjustments) locally on your system. It seems Jenkins does not like MSWs.

Reply:
As long as the relative tolerance is coarse enough the absolute tolerance can be arbitrary small. I got it through with an absolute tolerance of 2e-12... With that, I tested the boundary between success and failure for the relative tolerance, and got success at 3e-4, failure at 2e-4. So 1e-3 is a bit course, but I think it is not a good idea to go very close to the boundary, so it is ok to keep it at that.